stdlib/public/Concurrency/Task.swift

```diff
@@ -17,15 +17,23 @@ import Swift
 
 /// A unit of asynchronous work.
 ///
-/// An instance of `Task` always represents a top-level task. The instance
-/// can be used to await its completion, cancel the task, etc., The task will
-/// run to completion even if there are no other instances of the `Task`.
+/// When you create an instance of `Task`,
+/// you provide a closure that contains the work for that task to perform.
+/// Tasks can start running immediately after creation;
+/// you don't explicitly start or schedule them.
+/// After creating a task, you use the instance to interact with it ---
+/// for example, to wait for it to complete or to cancel in.
+/// It's not a programming error to discard a reference to a task
+/// without waiting for that task to finish or canceling it.
+/// A task runs whether or not you keep a reference to it.
+/// However, if you discard a task's handle, you give up the ability
+/// to wait for that task's result or cancel the task.
 ///
-/// `Task` also provides appropriate context-sensitive static functions which
-/// operate on the "current" task, which might either be a detached task or
-/// a child task. Because all such functions are `async` they can only
-/// be invoked as part of an existing task, and therefore are guaranteed to be
-/// effective.
+/// To support operations on the current task,
+/// which can be either a detached task or a child task,
+/// `Task` also exposes class methods like `yield()` and `currentPriority`.
+/// Because all such functions are asynchronous,
+/// they're always invoked as part of an existing task.
```

Comment:
`currentPriority` is not async so this needs some rewording.

currentPriority will try to get a priority from current task if called within one, and if not, it will try to make a best approximation of priority from the current thread priority.

Perhaps only mention yield here?

Reply:
Changed in 946c72cbc73 to just mention `yield`.